Package.swift

```diff
@@ -125,16 +137,52 @@ var targets: [PackageDescription.Target] = [
         ]),
 ]
 
+#if swift(>=5.8)
```

Comment:
This is a bit unfortunate. What 5.8 language features are we using that we can't use in 5.7? Is it only the `some` keyword? If so it would be great if we could just write this differently and make it support 5.7

Reply:
5.8 introduced a few buffer methods (SE-0370) which we are using. I could rewrite it to use memcpy

Reply:
https://github.com/apple/swift-http-types/pull/2